public_suffix_list.dat

```diff
@@ -11667,6 +11667,46 @@ zapto.org
 // Submitted by Matthew Brown <mattbrown@nyc.mn>
 nyc.mn
 
+// NymNom : http://nymnom.com/
+// Submitted by Dave McCormack <dave.mccormack@nymnom.com>
+nom.ae
+nom.ai
+nom.al
+nom.cl
+nom.gd
+nom.gl
+nom.gt
+nom.hn
+nom.im
+nom.li
+nom.mk
+nom.nu
+nom.pw
+nom.qa
+nom.rs
+nom.si
+nom.ug
+nom.uy
+nom.vc
+nom.vg
+nym.by
```

Comment:
These should be sorted by the right-most label (the TLD) to the left-most label (nom vs nym); it's presently sorted left-most to right-most.